config-dev.yaml

```diff
@@ -49,8 +49,16 @@ kafkaStreamsConfig:
     replication.factor: 1
     num.stream.threads: 2
 
+# WARNING! This config is NOT RECOMMENDED for production. Dev-use only
+# recommended overrides in production = replicationFactor: 3
```

Comment:
There are other uses of replication factor... Maybe this should be documented on all of them?

Reply:
i documented in 2 places, one is config-dev.yml and stream-registry-config.yaml

but not in the one used in test cases, as it would not be used in any runtime env